app/controllers/home_controller.rb

```diff
@@ -0,0 +1,71 @@
+class HomeController < ApplicationController
+  def index
+  end
+
+  def show
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/AbcSize: Assignment Branch Condition size for show is too high. [64/15]